src/dndbeyond/content-scripts/character.js

```diff
@@ -240,6 +240,7 @@ function rollItem(force_display = false, force_to_hit_only = false, force_damage
     if (!force_display && Object.keys(properties).includes("Damage")) {
         const item_full_name = $(".ct-item-pane .ct-sidebar__heading .ct-item-name,.ct-item-pane .ct-sidebar__heading .ddbc-item-name").text();
         let to_hit = properties["To Hit"] !== undefined && properties["To Hit"] !== "--" ? properties["To Hit"] : null;
+        let settings_to_change = {}
```

Comment:
the variable should be const, since it doesn't change (its content is modified, but not the variable itself)